packages/client/src/components/app/blocks/RowExplorer.svelte

```diff
@@ -0,0 +1,247 @@
+            description: cardDescription,
+            imageURL: cardImageURL,
+            horizontal: true,
+            buttonOnClick: [
+              {
+                parameters: {
+                  key: stateKey,
+                  type: "set",
+                  persist: null,
+                  value: `{{ [${listRepeaterId}].[_id] }}`,
+                },
+                "##eventHandlerType": "Update State",
+                id: generate(),
+              },
+            ],
+          }}
+          styles={{
+            normal: {
+              width: "auto",
+            },
+          }}
+        />
+      </BlockComponent>
+    </BlockComponent>
+    <BlockComponent
+      type="container"
+      order={1}
+      props={{
+        hAlign: "center",
+        vAlign: "middle",
+        size: "grow",
+        direction: "column",
+      }}
+      styles={{
+        custom: `
+          padding: 20px;
+          background-color: var(--spectrum-global-color-gray-50));
+          border: 1px solid var(--spectrum-global-color-gray-300);
+          border-radius: 4px;
+          flex: 4;
+          {{#if (or [state].[${stateKey}] [device].[mobile]) }}
+            display: none;
+          {{/if}}
+          `,
+      }}
+    >
+      <BlockComponent
+        type="icon"
+        order={0}
+        props={{
+          icon: "ri-list-check-2",
+          size: "ri-2x",
+          color: "var(--spectrum-global-color-gray-700)",
+        }}
+        styles={{
+          normal: {
+            "margin-bottom": "12px",
+          },
+        }}
+      />
+      <BlockComponent
+        type="text"
+        order={1}
+        props={{
+          text: "Select a row to view its fields",
+          color: "var(--spectrum-global-color-gray-700)",
+        }}
+      />
+    </BlockComponent>
+    <BlockComponent
+      type="container"
+      order={2}
+      props={{
+        hAlign: "center",
+        vAlign: "top",
+        size: "grow",
+        direction: "column",
+      }}
+      styles={{
+        custom: `
+          background-color: var(--spectrum-global-color-gray-50));
+          border: 1px solid var(--spectrum-global-color-gray-300);
+          border-radius: 4px;
+          padding: 20px;
+          overflow-y: scroll;
+          flex: 4;
+          {{#if (isFalsey [state].[${stateKey}]) }}
+            display: none;
+          {{/if}}
+          `,
+      }}
+    >
+      <BlockComponent
+        type="button"
+        order={0}
+        props={{
+          text: "← Back",
+          onClick: [
+            {
+              parameters: {
+                key: stateKey,
+                type: "set",
+                persist: null,
+                value: "",
+              },
+              "##eventHandlerType": "Update State",
+              id: generate(),
+            },
+          ],
+        }}
+        styles={{
+          custom: `
+            align-self: flex-end;
+            margin-bottom: 5px;
```

Comment:
I think we need a little more spacing here since it looks a bit cramped on mobile - maybe 16px?

As an aside - I'll be updating the form block soon to allow executing actions when you press buttons, which will actually help this block a lot too. It means we could automatically close the form on mobile when saving changes.